samza-core/src/main/java/org/apache/samza/processor/SamzaContainerController.java

```diff
@@ -105,7 +106,9 @@ public void startContainer(ContainerModel containerModel, Config config, int max
         localityManager,
         new JmxServer(),
         Util.<String, MetricsReporter>javaMapAsScalaMap(metricsReporterMap),
-        taskFactory);
+        taskFactory,
+        // TODO: need to use the correct local ApplicationRunner here
+        ApplicationRunner.getLocalRunner(config));
```

Comment:
Shouldn't this be getRemoteRunner? If I understand correctly, local runner is meant for standalone.